escrow/Cargo.toml

```
@@ -6,10 +6,8 @@ edition = "2021"
 license = "Apache-2.0"
 
 [dependencies]
-fuel-gql-client = { version = "0.6", default-features = false }
-fuel-tx = "0.9"
-fuels = "0.13"
-fuels-abigen-macro = "0.13"
+fuels = "0.14.1"
+fuels-abigen-macro = "0.14.1"
```

Prefer major.minor unless there is a specific requirement on an exact version.
```suggestion
fuels = "0.14"
fuels-abigen-macro = "0.14"
```